test/integration/util/kubelet/testdata/insecure-kubelet-compose.yaml

```diff
@@ -1,18 +1,20 @@
 version: '2.3'
 services:
   kubelet:
-    image: "datadog/docker-library:hyperkube_1_8_3"
-    command: "/hyperkube
-        kubelet
-        --cloud-provider=''
+    image: "datadog/docker-library:kubelet_1.25"
```

Comment:
you are mixing hyperkube 1.18 with kubelet 1.25. Is it ok?

Reply:
ok forget about my previous message. files are used in different checks

Reply:
It's not used in the same tests but yeah I could migrate to standard apiserver as well.

Reply:
You cannot bump to `1.25` has the `--insecure-bind-address` flag has been removed.